CHANGELOG.md

```diff
@@ -1,5 +1,104 @@
+- feat!: new required attributes `authorized_user_first_name` and `authorized_user_last_name` in `mongodbatlas_backup_compliance_policy` resource. [\#1655](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1655) ([marcosuma](https://github.com/marcosuma))
+
+**Enhancements**
+
+- feat: new `mongodbatlas_stream_instance` resource [\#1685](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1685) ([AgustinBettati](https://github.com/AgustinBettati))
+- feat: new `mongodbatlas_stream_instance` and `mongodbatlas_stream_instances` data sources [\#1689](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1689) [\#1701](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1701) ([AgustinBettati](https://github.com/AgustinBettati))
+- feat: new `mongodbatlas_stream_connection` resource [\#1736](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1736) ([AgustinBettati](https://github.com/AgustinBettati))
+- feat: new `mongodbatlas_stream_connection` and `mongodbatlas_stream_connections` data sources [\#1757](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1757) ([AgustinBettati](https://github.com/AgustinBettati))
+
+**Bug Fixes**
+
+- fix: Doesn't disable X.509 in the project when mongodbatlas\_x509\_authentication\_database\_user resource is deleted [\#1760](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1760) ([lantoli](https://github.com/lantoli))
+- fix: adjust root `project_id` attribute in `mongdbatlas_project_api_key` resource to optional [\#1664](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1664) ([AgustinBettati](https://github.com/AgustinBettati))
+- fix: define `project_assignment` block in `mongodbatlas_project_api_key` as required to avoid plugin crash [\#1663](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1663) ([AgustinBettati](https://github.com/AgustinBettati))
+- fix: Fixes cluster update when adding replication specs [\#1755](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1755) ([lantoli](https://github.com/lantoli))
+- fix: Provider crash when an invalid role name is specified to `mongodbatlas_project_api_key` [\#1720](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1720) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- fix: project delete function uses correct string representation of id [\#1733](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1733) ([oarbusi](https://github.com/oarbusi))
+
+**Bug Fixes for Terraform Version 1.0.8**
+
+- fix: Fixes some tests in search\_index test group in older TF versions [\#1758](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1758) ([lantoli](https://github.com/lantoli))
```

Comment:
Believe we are a missing few: 

serverless
cloud_provider_access
org_invitation
project_api_key
third_party_integration

more details in: https://mongodb.slack.com/archives/C04K3F1EGGK/p1702653073328139?thread_ts=1702651967.537649&cid=C04K3F1EGGK

Reply:
that's correct, sorry the PR titles don't cover all of them.

Reply:
perfect, defined resource level details here